modules/update-restricted-network-cli.adoc

```diff
@@ -0,0 +1,106 @@
+
+[id="update-restricted-network-cli_{context}"]
+= Update the restricted network cluster
+
+Verify the sig
+
+.Prerequisites
+
+* You have access to the mirror registry that you used to store the images that
+you used to install {product-title}.
+* You have the `ImageContentSourcePolicy` information for the
+* The the OpenShift Command-line Interface (CLI), commonly known as `oc`, is
+installed.
+* The `jq` package is installed.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Create an `ImageContentSourcePolicy` resource for your cluster:
++
+----
+$ cat test.yaml
+
+apiVersion: operator.openshift.io/v1alpha1 <1>
+kind: ImageContentSourcePolicy
+metadata:
+  name: example
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-release
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+----
+<1> Specify the entire specification that was showin in the output of the
+`oc adm mirror release` command that mirrored the images for this release.
+
+. Create the `ImageContentSourcePolicy` resource that you defined:
++
+----
+$ oc create -f test.yaml
+----
+
+. Start the cluster update:
++
+----
+$ oc adm upgrade --force --to-image <local_registry_host_name>:<local_registry_host_port>/ocp/release:<release_version> <1>
+----
+<1> For `<local_registry_host_name>` and `<local_registry_host_port>`, specify
+the values that describe your mirror registry host name and port. For
+`<release_version>`, specify the version that you want to upgrade to.
+
+. Review the cluster version status history to monitor the status of the update.
+It might take some time for all the objects to finish updating.
++
+----
+$ oc get clusterversion -o json|jq ".items[0].status.history"
+
+[
+  {
+    "completionTime": null,
+    "image": "quay.io/openshift-release-dev/ocp-release@sha256:9c5f0df8b192a0d7b46cd5f6a4da2289c155fd5302dec7954f8f06c878160b8b",
+    "startedTime": "2019-06-19T20:30:50Z",
+    "state": "Partial",
+    "verified": true,
+    "version": "4.2.1"
+  },
+  {
+    "completionTime": "2019-06-19T20:30:50Z",
+    "image": "quay.io/openshift-release-dev/ocp-release@sha256:b8307ac0f3ec4ac86c3f3b52846425205022da52c16f56ec31cbe428501001d6",
+    "startedTime": "2019-06-19T17:38:10Z",
+    "state": "Completed",
+    "verified": false,
+    "version": "4.2.0"
+  }
+]
+----
++
+The history contains a list of the most recent versions applied to the cluster.
+This value is updated when the CVO applies an update. The list is ordered by
+date, where the newest update is first in the list. Updates in the history have
+state `Completed` if the rollout completed and `Partial` if the update failed
+or did not complete.
++
+[IMPORTANT]
+====
+If an upgrade fails, the Operator stops and reports the status of the failing
```

Comment:
The cluster-version operator aborts that particular upgrade round, and starts over, attempting to apply it again.  Maybe just say:

> If an upgrade fails, ClusterVersion will include a condition explaining why.